### PR TITLE
Out of memory fixes and script cleanup

### PR DESCRIPTION
**Pull request by paulbatum**
With these changes I can now do 20 minute 100 EPS runs without getting out of memory errors. Also I updated the function scripts - the C# script did not compile and some of the JS scripts were out of sync.

### CODE REVIEW

**Comment by hforeste** [comment]
Reviewing...

File: ServerlessBenchmark/PerfResultProviders/AzurePerformanceResultProviderBase.cs [hunk 1]
```diff
@@ -44,39 +45,47 @@ protected int CalculateExecutionCount(string functionName, DateTime testStartTim
...
-                select new { TimeStamp = g.Key, Count = g.Count() };           
-
+                select new { TimeStamp = g.Key, Count = g.Count() };
+
             var actualStartTime = grouped.Min(x => x.TimeStamp);
             var actualEndTime = grouped.Max(x => x.TimeStamp);
             var countByTimestamp = grouped.ToDictionary(x => x.TimeStamp, x => x.Count);
 
-            var stringBuffer = new StringBuilder();
+            if (actualEndTime - actualStartTime > TimeSpan.FromHours(24))
```
Comment (KrzysiekKnigawka) [inline ServerlessBenchmark/PerfResultProviders/AzurePerformanceResultProviderBase.cs]: maybe it's a good idea to keep 24 as a constant somewhere

File: ServerlessBenchmark/PerfResultProviders/AzurePerformanceResultProviderBase.cs [hunk 2]
```diff
@@ -44,39 +45,47 @@ protected int CalculateExecutionCount(string functionName, DateTime testStartTim
...
+            {
+                throw new Exception(string.Format("Aborting calculation. The max time range is 24 hours. StartTime: {0}, EndTime: {1}.", actualStartTime, actualEndTime));
+            }
+
             var throughputList = new List<int>();
+            var fileName = string.Format("{0}-Throughput.txt", Guid.NewGuid().ToString());
```
Comment (KrzysiekKnigawka) [inline ServerlessBenchmark/PerfResultProviders/AzurePerformanceResultProviderBase.cs]: adding common prefix to output file could help find the file afterwards

File: ServerlessBenchmark/PerfResultProviders/AzurePerformanceResultProviderBase.cs [hunk 3]
```diff
@@ -109,20 +119,26 @@ protected double CalculateHostConcurrency(string functionName, DateTime testStar
...
             }
 
-            var stringBuffer = new StringBuilder();
-
             var orderedLogs = processedLogs.OrderBy(l => l.Timestamp);
             var actualStartTime = orderedLogs.First().Timestamp;
             var actualEndTime = orderedLogs.Last().Timestamp;
-            for (var timeStamp = actualStartTime; timeStamp < actualEndTime; timeStamp = timeStamp.AddSeconds(1))
+
+            if (actualEndTime - actualStartTime > TimeSpan.FromHours(24))
```
Comment (KrzysiekKnigawka) [inline ServerlessBenchmark/PerfResultProviders/AzurePerformanceResultProviderBase.cs]: you can usa same const for 24 hour time

**Pull request merged**